pyproject.toml

```diff
@@ -125,9 +125,9 @@ warn_untyped_fields = true
 skip = './docs/_build, ./build, ./src/stravalib/tests/resources'
 
 [tool.ruff]
-select = [
+lint.select = [
```

Comment:
Ruff raises warnings locally. this fixes those and adheres to the newer release.